assets/js/ajax-filters.js

```diff
@@ -175,15 +183,13 @@ jQuery( document ).ready( function ( $ ) {
 		} );
 	} );
 
-	$( '#search_keywords, #search_location, .job_types :input, #search_categories, .job-manager-filter' ).change( function() {
-		var target   = $( this ).closest( 'div.job_listings' );
-		target.triggerHandler( 'update_results', [ 1, false ] );
-		job_manager_store_state( target, 1 );
-	} )
+	//end of the initial update_results listener
 
-	.on( "keyup", function(e) {
+	// Changed this to just listen for enter button instead of listening for change event which was calling the loop twice.
```

Comment:
Yes you are right, my comment is misleading, it was not the change event that was causing the error but the invocation of `job_manager_store_state` on line [181](https://github.com/Automattic/WP-Job-Manager/blob/master/assets/js/ajax-filters.js#L181), I will replace the change event listener and instead comment out the invocation and rely instead on line [31](https://github.com/lindsaymacvean/WP-Job-Manager/blob/master/assets/js/ajax-filters.js#L31) to call the HTML5 state management function only once